frontend/src/components/Map/DeleteMapButton.tsx

```
@@ -0,0 +1,23 @@
+  text: string;
+};
+
+const DeleteMapButton = ({ mapId, text }: DeleteMapButtonProps) => {
+  const deleteMutation = useDeleteMapMutation();
+
+  const onClick = () => {
+    deleteMutation.mutate(mapId);
+  };
+  return (
+    <button onClick={onClick} className="text-xs text-c_placeholder_gray">
+      {text}
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)